racket-mode.el

```diff
@@ -108,6 +108,10 @@ http://www.gnu.org/licenses/ for details.")
      ["Last S-Expression" racket-expand-last-sexp]
      "---"
      ["Again" racket-expand-again])
+    ("Tools"
+     ["Profile" racket-profile]
+     ["Check Syntax" racket-check-syntax-mode]
+     ["Error Trace" racket-errortrace])
```

Comment:
> On Feb 24, 2015, at 19:09, Greg Hendershott <notifications@github.com> wrote:
> 
> Thanks for doing this. One hitch: The last/third item should be deleted. There's no racket-errortrace function.

oops! Fixed. I wound up writing this into the installed package because I used elisp/emacs to find racket-run.